### PR TITLE
Base plotter

### LINKED ISSUE #13: Build base plotter service

### PR DESCRIPTION
**Pull request by antoniomf97**
Added base plotter for a simple 2d plot.

### CODE REVIEW

File: services/plotter/models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+
+class Configurations(BaseModel):
```
Comment (easyg24) [inline services/plotter/models.py:4]: Later on we must define the data model

File: services/plotter/tmp/test.png
Comment (easyg24) [inline services/plotter/tmp/test.png:1]: Maybe not needed to save this

**Review by easyg24 — APPROVED** [review]
Approved for now. Later we recall this comments

File: services/plotter/tmp/test.png
Comment (antoniomf97) [reply services/plotter/tmp/test.png:1]: For now we can save for testing, later we delete.

**Pull request merged**